UMutils.py

```diff
@@ -453,11 +443,13 @@ def scoring_goldstandard(row):
         file = file[file.answer_uuid.str.len() > 3]
 
         # these are the only relevant columns for scoring for now, notice highlight data is not included here
-        cols = ["answer_uuid", "question_Number", "agreed_Answer"]
+        cols = ["answer_uuid", "question_Number", "agreed_Answer", "source_task_uuid"]
```

Comment:
I think this is a bug now, selecting the same answer in different tasks should be treated as two different questions?

Reply:
Notably this breaks the scoring that depends on the grouping.

Reply:
I don't fully understand the bug yet, will have to look into it more. Otherwise the other organizational changes look good.